services/catalog/src/simcore_service_catalog/models/domain/dag.py

```diff
@@ -15,17 +13,18 @@ class DAGBase(BaseModel):
     )
     version: str = Field(..., regex=VERSION_RE, example="1.0.0")
     name: str
-    description: Optional[str]
-    contact: Optional[LowerCaseEmailStr]
+    description: str | None
+    contact: LowerCaseEmailStr | None
 
 
 class DAGAtDB(DAGBase):
     id: int
-    workbench: Json[dict[str, Node]]  # pylint: disable=unsubscriptable-object
+    # pylint: disable=unsubscriptable-object
+    workbench: Json[dict[str, Node]]  # type: ignore
```

Comment:
I think you already know that pylint: disable on the line above will apply to the entire block of code from that point onwards.

I don't ket why the type: ignore is required, since there already is a type there

Reply:
``ignore`` is required because of ``Json``'s type. In the code of pydantic version we have it is written as

```python
if TYPE_CHECKING:
    Json = str

```